client/web/antrea-ui/src/components/login.tsx

```diff
@@ -48,8 +48,8 @@ export default function Login(props: { setToken: (token: string) => void }) {
         <form onSubmit = {handleSubmit(onSubmit)}>
             <CdsFormGroup layout="horizontal">
                 <CdsInput>
-                    <label>Username</label>
-                    <input {...register("username", {
+                    <label htmlFor="username-input">Username</label>
```

Comment:
Are these `for` properties only used for testing? I saw CdsInput can associate the label and input in the browser. But seems it does not work in testing. Not sure whether it is because CDS components use lit underneath.

Reply:
Let me double-check. I see that in the clarity react unit tests, they are able to get inputs by label without explicitly adding this attribute: https://github.com/vmware-clarity/core/blob/main/projects/react/src/input/index.test.tsx

Reply:
Good catch. The `for` attribute is not required as it will be added automatically by clarity. However, in tests, the async version of the `ByLabelText` function needs to be used, since the attributes will be added asynchronously during rendering. I updated the test cases accordingly and remove the manual `htmlFor` / `id` attributes for the frontend React code.